xfid_test.go

```diff
@@ -991,23 +990,19 @@ func TestXfidwriteQWeventExecuteSend(t *testing.T) {
 	w.nopen[QWevent]++
 	defer func() { w.nopen[QWevent]-- }()
 	w.tag = Text{
-		w: w,
-		file: &File{
-			b:    RuneArray("Send"),
-			text: []*Text{&w.tag},
-		},
+		w:       w,
+		file:    MakeObservableEditableBuffer("", RuneArray("Send")),
 		fr:      &MockFrame{},
 		display: d,
 	}
+	w.tag.file.AddObserver(&w.tag)
```

Comment:
Did you need to add this line? I don't see the point where we add an observer.

Reply:
No, my bad